lib/_settings.scss

```diff
@@ -40,24 +40,25 @@ $gel-type-no-touch-group: 'group-d';
 
 /// Output configuration
 //
-// The default namespace applied to all typography classes output is inherits
-// from the default gel-namespace: `gel-`.
+// By default the typography component has no output and just exposes a
+// series of Sass mixins and tools. Enabling this option will output
+// a series of typography classes to be used within your markup.
 //
-// @link http://bit.ly/1Z6hPfd
-//
-// @type String
+// @type Bool
 //
 $gel-type-enable--markup-output: false !default;
 
-// Which typography group contains our non-touch typography styles
+// Barlesque will set the correct font-family on the <body> tag but
```

Comment:
Given this is an open source project, we should make it clear that `Barlesque` is the BBC's header and footer navigation. If someone is using this library on a non BBC project what they need to do. This should also be documented in the `README.md` docs.